front/app/containers/Admin/reporting/components/ReportBuilder/Templates/ProjectTemplate.tsx

```diff
@@ -51,63 +58,62 @@ const ProjectTemplate = ({ reportId, projectId }: Props) => {
     ? getProjectPeriod(phases.data)
     : { startAt: undefined, endAt: moment().format('YYYY-MM-DD') };
 
+  if (!appConfigurationLocales) return null;
+
+  const toMultiloc = (message: MessageDescriptor) => {
```

Comment:
This one could be useful for the other content builders, as well, if we decide we want to have default content in some components. Could we make it a util to be reused?

Reply:
I don't think we can really make this more reusable as a utility. It depends on `appConfigurationLocales` and `formatMessageWithLocale`. So we would need to make it a hook.